src/main/java/persistence/sql/entity/manager/EntityManagerImpl.java

```diff
@@ -60,6 +62,17 @@ public <T> T find(final Class<T> clazz, final Object id) {
         return entity;
     }
 
+    private <T> T getEntityLoader(final Class<T> clazz, final Object id) {
+        EntityEagerLoader entityEagerLoader = metaModel.getEntityEagerLoader(clazz);
+        EntityLoader entityLoader = metaModel.getEntityLoader(clazz);
+
+        if(entityEagerLoader.isEagerFetchType(clazz)) {
```

Comment:
아하 여기로 이동시켜 보셨군요! 

개인적인 관점으로는, `entityEagerLoader.isEagerFetchType`으로 판단하는게 어색해보여요.
왜냐면, `EntityEagerLoader`의 내부상태를 이용하지 않고있기때문에 적절한 책임인가? 의문이 들었어요.

저는 최대한 내부상태가 있는 객체에게 물어보는게 재활용성이나 객체의 명확한 책임을 정의 내리는것에 좋더라구요.

그런데, 저도 딱히 어디로 갈지 잘 모르겠어서 우선은 여기에 그냥 놔둘것 같아요 아래처럼요 ㅎ
```suggestion
final EntityMappingTable entityMappingTable = EntityMappingTable.from(clazz);
 if(entityMappingTable.hasFetchType(FetchType.EAGER)) {
```

Reply:
EntityMappingTable을 EntityManager 쪽에 만들기 애매해서 EntityEagerLoader 에서 관리를 했는데 이게 더 select에대한 역할을 확실히 할 수 있곘군요